std/array.d

```diff
@@ -350,7 +344,7 @@ See_Also: $(REF Tuple, std,typecons)
  */
 
 auto assocArray(Range)(Range r)
-if (isInputRange!Range)
+    if (isInputRange!Range)
```

Comment:
It's an automatic revert & the spaces before the if constraint are the common Phobos style